docker-storage-setup.sh

```diff
@@ -632,7 +632,7 @@ canonicalize_block_devs() {
     # Looks like we allowed just device name (sda) as valid input. In
     # such cases /dev/$dev should be a valid block device.
     if [ -h "$dev" ];then
-      dev=$(readlink -f $dev)
+      dev=$(readlink -e $dev)
```

Comment:
Also you can modify your existing patch and do a force push. Don't have to create incremental patch for taking care of review feedback.